earthio/tests/test_netcdf.py

```diff
@@ -28,9 +28,8 @@ def test_read_meta():
 
 
 def _validate_array_test_result(ds):
-    sample = ds.HQobservationTime
-    mean_y = np.mean(sample.y)
-    mean_x = np.mean(sample.x)
+    mean_y = np.mean(ds.y)
+    mean_x = np.mean(ds.x)
```

Comment:
Can you explain the significance of this change?  Is it adapting to a change in ds that moved x and y down into ds itself?

Reply:
I'm still play with this, I'll WIP to this PR.  I'm not convinced this change is needed.